grafana_objects/client_grafana_objects_create_update.go

```diff
@@ -0,0 +1,70 @@
+package grafana_objects
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	logzio_client "github.com/logzio/logzio_terraform_client"
+	"github.com/logzio/logzio_terraform_client/client"
+)
+
+type CreateUpdatePayload struct {
+	Dashboard map[string]interface{} `json:"dashboard"`
+	FolderId  int                    `json:"folderId"`
+	FolderUid int                    `json:"folderUid"`
+	Message   string                 `json:"message"`
+	Overwrite bool                   `json:"overwrite"`
+}
```

Comment:
First, please move the structs that represent a request and a response to `client_grafana_objects.go`.
Second, ss a general rule of thumb, if you can use structs to represent your JSON data, you should use them. The only good reason to use maps would be if it were not possible to use structs due to the uncertain nature of the keys or values in the data.
Please change the map, and create a dashboard object with the relevant fields, then use it as a nested field in CreateUpdatePayload etc...

Reply:
done, not sure abut `Test_ValidateTimepickerStruct`, `Test_ValidateTemplatingStruct` and `Test_ValidateDashboardObjectsStruct` though